docs/web-chat/api.mdx

```diff
@@ -137,6 +137,151 @@ You should call this API before a conversation starts. In other words, it works
 
 </Admonition>
 
+## Login {#login}
+
+The Login API is introduced along with the [new Identity system](/web-chat/users/#managing-users-with-the-identity-system).
+You can refer to the linked section for details.
+
+## Logout {#logout}
+
+The Logout API is introduced along with the [new Identity system](/web-chat/users/#managing-users-with-the-identity-system).
+You can refer to the linked section for details.
+
+
+## Update global user data {#update-global-user-data}
+
+This event is used in conjunction with the [new Identity system](/web-chat/users/#managing-users-with-the-identity-system) and is used to update the global data associated with a logged-in user.
+
+The following key/value pairs are accepted in this API call:
+```ts
+interface CustomUserFields {
+  [key: string]: string
+}
+
+interface GlobalUserData {
+     // Note that all primitive values must be coerced to string
+     first_name?: "string",
+     last_name?: "string",
+     full_name?: "string",
+     display_name?: "string",
+     last_country?: "string",
+     last_city?: "string",
+     age?: "string",
+     lifetime_value?: "string",
+     user_persona?: "string",
+     user_vip_segment?: "string",
+     user_support_status?: "string"
+     last_active_date?: "string" // epoch time in ms/seconds
+     accepted_t_and_c?: "string" // true/false converted to string
+     preferred_language?:"string"
+     full_privacy_enabled?: "string"
```

Comment:
In the Login API, Full Privacy Flag is present, are we allowing to modify from global fields as well?